spec.html

```diff
@@ -32049,8 +32049,8 @@ <h1>Runtime Semantics: RegExpInitialize ( _obj_, _pattern_, _flags_ )</h1>
             1. Else, let _P_ be ? ToString(_pattern_).
             1. If _flags_ is *undefined*, let _F_ be the empty String.
             1. Else, let _F_ be ? ToString(_flags_).
-            1. If _F_ contains any code unit other than `"g"`, `"i"`, `"m"`, `"s"`, `"u"`, or `"y"` or if it contains the same code unit more than once, throw a *SyntaxError* exception.
-            1. If _F_ contains `"u"`, let _BMP_ be *false*; else let _BMP_ be *true*.
+            1. If _F_ contains any code unit other than `g`, `i`, `m`, `s`, `u`, or `y` or if it contains the same code unit more than once, throw a *SyntaxError* exception.
+            1. If _F_ contains `u`, let _BMP_ be *false*; else let _BMP_ be *true*.
```

Comment:
Comparing the two lines you deemed worth changing to the two lines here, I find myself asking: _even when a distinction is being made, is it being made meaningfully?_

I wonder if it would be better to close this PR entirely and create an issue to re-evaluate usage of "code unit" and "code point" throughout the spec...?

Reply:
Note: If we do keep the status quo of referring to RegExp flags with strings across the board, then we should instead fix the singular place where we're not doing that: the `y` in [this note](https://tc39.es/ecma262/#sec-assertion).

Reply:
Well, there are two ways in which RegExp flags manifest:
- as code points in source text; and
- as code units in String values (arising either from the stringification of the above source text, or via the 'flags' parameter to the RegExp constructor).

Lines 12999 and 30942 are talking about flags as source text; lines 32052-3 are talking about flags in a String value.

> I wonder if it would be better to close this PR entirely and create an issue to re-evaluate usage of "code unit" and "code point" throughout the spec...?

If the point of the issue would be to ask whether the distinction between "code unit" and "code point" is meaningful, then I suspect you'll just get the answer "yes, it is". Don't let me stop you, though.